packages/fx-core/src/plugins/resource/appstudio/manifest/src/index.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { TeamsAppManifest } from "@microsoft/teamsfx-api";
+
+class Manifest {
+  /**
+   * Load manifest from manifest.json
+   * @param filePath path to the manifest.json file
+   */
+  async load(filePath: string): Promise<TeamsAppManifest> {
+    return new TeamsAppManifest();
+  }
+
+  /**
+   * Save manifest to .json file
+   * @param filePath path to the manifest.json file
+   * @param manifest
+   */
+  async save(filePath: string, manifest: TeamsAppManifest): Promise<any> {}
```

Comment:
Should we make it static since manifest is passed in. Or perhaps it's more OOP-ish to have this signature `save(filePath: string): Promise<void>`, which is more consistent with `fs.writeJSON`. 
Besides, we should probably add more Exception types, and document them in comments for bettern error handling